dapr/proto/runtime/v1/dapr.proto

```diff
@@ -60,6 +63,29 @@ message GetStateRequest {
   common.v1.StateOptions.StateConsistency consistency = 3;
 }
 
+// GetBulkStateRequest is the message to get a list of key-value states from specific state store.
+message GetBulkStateRequest {
+  // The name of state store.
+  string store_name = 1;
+
+  // The keys to get
+  repeated string keys = 2;
+}
+
+// GetBulkStateResponse is the response conveying the list of state values.
+message GetBulkStateResponse {
+  repeated BulkStateItem items = 1;
+}
+
+// GetStateResponse is the response conveying the state value and etag.
+message BulkStateItem {
+  // state item key
+  string key = 1;
+
+  // The byte array data
+  bytes data = 2;

```

Comment:
missing string etag = 3; ? As per comment, it should return etag...

If so, might need to update the comment to include "key" as well...

Reply:
Addressed in latest commit.